subsys/partition_manager/pm.yml.emds

```diff
@@ -0,0 +1,6 @@
+#include <autoconf.h>
+
+emds_storage:
+  placement: {before: [end]}
+  size: CONFIG_PM_PARTITION_SIZE_EMDS_STORAGE
```

Comment:
Are `CONFIG_PM_PARTITION_SIZE_EMDS_STORAGE` and 0x1000 the same?

Reply:
The default value for CONFIG_PM_PARTITION_SIZE_EMDS_STORAGE is partition-size. This is done inside "Kconfig.template.partition_size". So by define you own CONFIG_PM_PARTITION_SIZE_EMDS_STORAGE can give you something else.